### PR TITLE
WM-1586 Submission details main fields 

### PR DESCRIPTION
**Pull request by kpierre13**

### CODE REVIEW

File: src/pages/SubmissionDetails.js [hunk 1]
```diff
@@ -36,34 +51,90 @@ export const SubmissionDetails = ({ submissionId }) => {
...
       ]),
-      div([
-        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`])
-      ]),
+      div({ style: { marginLeft: '4em' } }, [
+        h(TextCell, [(h(Link, { onClick: () => Nav.goToPath('submission-history') }, ['Submission History'])), ' >', ` Submission ${submissionId}`]),
+        h2(['workflow: ', getSpecificMethod[0]?.name]),
+        h3(['Submission date: ', specifyRunSet[0] && Utils.makeCompleteDate(specifyRunSet[0].submission_timestamp)]),
+        h3(['Duration: ', specifyRunSet[0] && Utils.customFormatDuration(duration(specifyRunSet[0]))]),
+        h3(['Submitted by: Batch Teams (HARDCODED)'])
```
Comment (cjllanwarne) [inline src/pages/SubmissionDetails.js:111]: We should probably make a ticket to record (and retrieve) this value

File: src/pages/SubmissionDetails.js [hunk 2]
```diff
@@ -36,34 +51,90 @@ export const SubmissionDetails = ({ submissionId }) => {
...
       div({
         style: {
           marginTop: '1em', height: tableHeight({ actualRows: paginatedPreviousRuns.length, maxRows: 12.5, heightPerRow: 250 }), minHeight: '10em'
         }
       }, [
+        div([h2(['Workflows'])]),
+        div([h3(['Filter by: '])]),
+        h(Select, {
+          isDisabled: false,
+          'aria-label': 'Select a data table',
```
Comment (cjllanwarne) [inline src/pages/SubmissionDetails.js:132]: `'Select a data table'` doesn't look right?

File: src/pages/SubmissionDetails.js [hunk 3]
```diff
@@ -36,34 +51,90 @@ export const SubmissionDetails = ({ submissionId }) => {
...
+        padding: '1rem 3rem'
+      }
+    }, [
       div({
         style: {
           marginTop: '1em', height: tableHeight({ actualRows: paginatedPreviousRuns.length, maxRows: 12.5, heightPerRow: 250 }), minHeight: '10em'
         }
       }, [
+        div([h2(['Workflows'])]),
+        div([h3(['Filter by: '])]),
```
Comment (cjllanwarne) [inline src/pages/SubmissionDetails.js:129]: When I tried this, this drop down list was empty. 
image

Comment (cjllanwarne) [reply src/pages/SubmissionDetails.js:129]: If that's deliberate, is there a Jira ticket for adding options?

File: src/pages/SubmissionDetails.js [hunk 4]
```diff
@@ -77,48 +148,30 @@ export const SubmissionDetails = ({ submissionId }) => {
               {
                 size: { basis: 350 },
                 field: 'record_id',
-                headerRenderer: () => h(Sortable, { sort, field: 'record_id', onSort: setSort }, ['Record Entry']),
+                headerRenderer: () => h(Sortable, { sort, field: 'record_id', onSort: setSort }, ['Workflow name']),
```
Comment (cjllanwarne) [inline src/pages/SubmissionDetails.js:151]: As we discussed with Sara, this should probably be "<Whatever the record type is> ID" (eg "Sample ID" for samples, "Participant ID" for participants... "FOO ID" for "FOO"s...

File: src/pages/SubmissionDetails.js [hunk 5]
```diff
@@ -136,27 +189,19 @@ export const SubmissionDetails = ({ submissionId }) => {
...
-                }
-              },
-              {
-                size: { basis: 250, grow: 0 },
-                field: 'logs',
-                headerRenderer: () => 'Logs',
-                cellRenderer: ({ rowIndex }) => {
-                  return div({ style: { width: '100%', textAlign: 'center' } }, [
-                    h(Link, { disabled: true, onClick: () => setViewInputsId(rowIndex) }, ['View workflow log file'])
+                    h(Link, { style: { display: 'inline-block', textDecoration: 'underline' }, onClick: () => setViewInputsId(rowIndex) }, [paginatedPreviousRuns[rowIndex].run_id])
```
Comment (cjllanwarne) [inline src/pages/SubmissionDetails.js:197]: On click could copy the deleted action above, ie `{ onClick: () => { Nav.goToPath('workflow-dashboard', { workflowId: paginatedPreviousRuns[rowIndex].engine_id }) }, style: { fontWeight: 'bold' } },`

**Review by cjllanwarne — COMMENTED** [review]
Looked like maybe some overlap with the table ticket, so I commented on those changes too. If that's just a rebase thing, feel free to ignore or copy those comments across to your other branch

**Review by cjllanwarne — APPROVED** [review]
👍 on the "main fields", and a few minor comments on the table

File: src/pages/SubmissionDetails.js (hunk 2 above)
Comment (kpierre13) [reply src/pages/SubmissionDetails.js:132]: I think this is changed in the previous ticket. This ticket was branched off of that one.

**Review by mspector — APPROVED** [review]

**Pull request merged**